scipy/stats/_multivariate.py

```diff
@@ -2378,6 +2378,11 @@ class invwishart_gen(wishart_gen):
     %(_doc_default_callparams)s
     %(_doc_random_state)s
 
+    Raises
+    ------
+    scipy.linalg.LinAlgError
+        If the scale matrix `scale` is not positive definite.
```

Comment:
```suggestion
        If the scale matrix `scale` is not numerically positive definite.
```
The example in the numpydoc style guide for "Raises" is `LinAlgException`: "If the matrix is not _numerically_ invertible.", so I could see an argument for making this change. I